framework/src/main/java/com/jfcbuilder/builders/XYDrawableBuilder.java

```diff
@@ -61,62 +64,126 @@ public static XYDrawableBuilder get() {
     return new XYDrawableBuilder();
   }
 
+  /**
+   * Sets the drawable's x-axis data coordinate.
+   * 
+   * @param x The data coordinate to set
+   * @return Reference to this builder instance for method chaining
+   */
   public XYDrawableBuilder x(double x) {
     this.x = x;
     return this;
   }
 
+  /**
+   * Gets the x-axis data coordinate that is set.
+   * 
+   * @return The data coordinate value
+   */
   public double x() {
     return x;
   }
 
+  /**
+   * Sets the drawable's y-axis data coordinate.
+   * 
+   * @param y The data coordinate to set
+   * @return Reference to this builder instance for method chaining
+   */
   public XYDrawableBuilder y(double y) {
     this.y = y;
     return this;
   }
 
+  /**
+   * Gets the y-axis data coordinate that is set.
+   * 
+   * @return The data coordinate value
+   */
   public double y() {
     return y;
   }
 
+  /**
+   * Sets the width of the area into which to draw the {@link org.jfree.chart.ui.Drawable}.
+   * 
+   * @param w The width in data-space units
```

Comment:
XYDrawableAnnotation width and height seem to be in Java2D space. See the `draw()` method:
https://github.com/jfree/jfreechart/blob/v1.5.3/src/main/java/org/jfree/chart/annotations/XYDrawableAnnotation.java#L151